src/components/Footer/Footer.js

```diff
@@ -175,103 +170,92 @@ const Footer = ({
+              <Text style={[styles.text, styles.textDM]}>DM</Text>
+            </View>
+          </TouchableOpacity>
+        ) : (
+          <TouchableOpacity testID="shareBtn" onPress={onPressShare}>
+            <View style={styles.btnShare}>
+              <MemoIc_share height={20} width={21} />
+            </View>
+          </TouchableOpacity>
+        )}
         {disableComment ? (
-          <View style={styles.totalCommentContainer}>
+          <View testID="disableComment">
+            <View style={styles.btnComment(totalComment || 0)}>
+              <IconCommentArrow color={COLORS.grey410} />
+            </View>
             <Text style={styles.text}>{totalComment}</Text>
           </View>
         ) : (
           <TouchableOpacity
+            testID="availableComment"
             disabled={item?.isBlurredPost}
-            style={{flex: 1}}
-            onPress={onPressComment}>
-            <View style={styles.totalCommentContainer}>
-              <Text style={styles.text}>{totalComment}</Text>
+            onPress={onPressComment}
+            style={{flexDirection: 'row', alignItems: 'center'}}>
+            <View style={styles.btnComment(totalComment || 0)}>
+              <IconCommentArrow color={COLORS.grey410} />
             </View>
+            <Text style={styles.text}>{totalComment}</Text>
           </TouchableOpacity>
         )}
-        <View style={styles.rightGroupContainer}>
-          {showScoreButton ? (
-            <TouchableOpacity disabled={item?.isBlurredPost} onPress={onPressScore}>
-              <Memoic_globe
-                color={item?.isBlurredPost ? COLORS.gray : undefined}
-                height={20}
-                width={20}
-              />
-            </TouchableOpacity>
-          ) : (
-            <></>
-          )}
-          {handleBlockUi()}
+        {handleBlockUi()}
+        <View style={{flexDirection: 'row', alignItems: 'center'}}>
           <TouchableOpacity
             testID="downVoteBtn"
             disabled={loadingVote || item?.isBlurredPost}
-            style={styles.btn}
             onPress={onPressDownVote}>
-            <View style={styles.btnDownvote}>
+            <View>
               {statusVote === 'downvote' ? (
                 <View testID="downvoteOn">
-                  <MemoIc_arrow_down_vote_on
-                    color={item?.isBlurredPost ? COLORS.gray : undefined}
-                    width={20}
-                    height={18}
-                  />
+                  <ArrowDownvoteOn color={item?.isBlurredPost ? COLORS.gray : undefined} />
                 </View>
               ) : (
                 <View testID="downvoteOff">
-                  <MemoIc_arrow_down_vote_off
-                    color={item?.isBlurredPost ? COLORS.gray : undefined}
-                    width={20}
-                    height={18}
-                  />
+                  <ArrowDownvoteOff color={item?.isBlurredPost ? COLORS.gray : undefined} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There are significant changes in the return statement of the Footer component. It appears that the layout has been redesigned with new styles and components. The `isShortText` prop is used to conditionally render different layouts. The voting buttons have been updated to use the new SVG components. The comment button now uses the `IconCommentArrow` component instead of the previous one. Ensure that these changes align with the intended design.
```diff
- <View style={[styles.rowSpaceBetween, styles.container]}>
+ <View style={styles.container(isShortText)}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->